pages/pipelines/managing_log_output.md.erb

```diff
@@ -143,3 +143,72 @@ For example, if you have environment variable `MY_SECRET="topsecret"`and you run
 <h3 class="Docs__note__heading">Setting environment variables</h3>
 <p>Note that if you <emphasis>set</emphasis> or <emphasis>interpolate</emphasis> a secret environment variable in your <code>pipeline.yml</code> it is not redacted, but doing that is <a href="/docs/pipelines/secrets#anti-pattern-storing-secrets-in-your-pipeline-dot-yml">not recommended</a>.</p>
 </div>
+
+## Private build log archive storage
+
+By default, build logs are stored in encrypted form in Buildkite's Amazon S3 buckets, but you can also store the archived build logs in your private AWS S3 bucket.
+
+<div class="Docs__note">
+<p>This feature is only available to <a href="https://buildkite.com/pricing">Buildkite Enterprise</a> users.</p>
+</div>
+
+To set up a private build log archive storage:
+
+1. Create an Amazon S3 bucket in *us-east-1* location (the only region currently supported).
+1. Provide *read* and *write* access permission policy for the Buildkite's AWS account `032379705303`. Here's an example policy that contains an Amazon S3 bucket configuration with Buildkite's account number in it. Configure your Amazon S3 bucket in a similar way:
+
```

Comment:
What I mean here is explicitly saying, 

replace `my-bucket` and `my-prefix` with your bucket info.

Reply:
On it!